13-Tenant Security Solution Setup/Scripts/RemediationScripts/Remediate-ConfigASCTier.ps1

```diff
@@ -0,0 +1,389 @@
+function Pre_requisites
+{
+    <#
+    .SYNOPSIS
+    This command would check pre requisities modules.
+    .DESCRIPTION
+    This command would check pre requisities modules to perform remediation.
+	#>
+
+    Write-Host "Required modules are: Az.Resources, Az.Security, Az.Account" -ForegroundColor Cyan
+    Write-Host "Checking for required modules..."
+    $availableModules = $(Get-Module -ListAvailable Az.Resources, AzureAD, Az.Accounts)
```

Comment:
Why we are not checking for Az.Security

Reply:
Is AzureAD required here.

Reply:
Added check for Az.Security and removed AzureAD